libs/sdk-ui-dashboard/src/model/state/catalog/catalogSelectors.ts

```diff
@@ -43,13 +43,19 @@ export const selectCatalogDateDatasets = createSelector(selectSelf, (state) => {
     return state.dateDatasets ?? [];
 });
 
+/**
+ * @internal
+ */
+export const selectCatalogDateAttributes = createSelector(selectCatalogDateDatasets, (dateDatasets) => {
+    return flatMap(dateDatasets ?? [], (dd) => dd.dateAttributes);
```

Comment:
Nitpick: the `?? []` is not needed, `selectCatalogDateDatasets` does that already.